docs/constants/navigation.js

```diff
@@ -319,6 +319,7 @@ const general = [
     ]),
     makeGroup('Reference', [
       makePage('eas-update/migrate-from-classic-updates.mdx'),
+      makePage('distribution/custom-updates-server.mdx'),
```

Comment:
```suggestion
      makePage('eas-update/custom-updates-server.mdx'),
```

Should we also change the URL prefix since we move this page, so it matches the organizaton we have?

This would also require adding a redirect, and ideally replacing old URL usages in other docs.

Not fully sure if this is required, or just a good practice, let's wait for Aman's take on that. 😉

Reply:
I agree we should update the URL prefix and add a redirect. Can also add a redirect in a follow up PR and check for replacing old URL usage in other guides.